### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by timheuer**
Fix typo on VS version

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/project-system/pull/7779)

### CODE REVIEW

File: docs/repo/property-pages/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿# Project Property Pages
 
-This documentation details the updated Project Properties UI and associated back end, added to Visual Studio in 2021.
```
Comment (jjmew) [inline docs/repo/property-pages/README.md:3]: I think this comment is correct. This is not talking about the version for Visual Studio but the timeframe of when it was added

File: docs/repo/property-pages/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+This documentation details the updated Project Properties UI and associated back end, added to Visual Studio in 2022.
```
Comment (tmeschter) [inline docs/repo/property-pages/README.md:3]: Talking about the product version is probably more useful than the year but we want to be clear on which one we mean.

```suggestion
This documentation details the updated Project Properties UI and associated back end, first released in Visual Studio 2022.
```

File: docs/repo/property-pages/README.md (hunk 1 above)
Comment (timheuer) [reply docs/repo/property-pages/README.md:3]: It's confusing as it's read -- we never talk about timeframes we talk about when/what products they are introduced.

Comment (jjmew) [reply docs/repo/property-pages/README.md:3]: The way the updated version reads makes sense

**Review by MiYanni — APPROVED** [review]

**Review by drewnoakes — APPROVED** [review]

**Review by jjmew — APPROVED** [review]
:shipit:

**Pull request merged**